frontend/server/api/auth/[...].ts

```diff
@@ -0,0 +1,81 @@
+// file: ~/server/api/auth/[...].ts
+import CredentialsProvider from 'next-auth/providers/credentials'
+import { NuxtAuthHandler } from '#auth'
+
+
+async function getCustomer(email:string) {
+  return await $fetch('/api/auth/customer', {
+    method: 'POST',
+    body: {
+      email: email,
+    },
+  });
+}
+
+export default NuxtAuthHandler({
+  providers: [
+    // @ts-expect-error You need to use .default here for it to work during SSR. May be fixed via Vite at some point\
```

Comment:
We don't need it then. SSR is disabled.

Reply:
Code producers errors without this line. I think we can keep this as is